extras/speaker-recognition/src/simple_speaker_recognition/api/service.py

```
@@ -97,6 +103,22 @@ async def lifespan(app: FastAPI):
+    try:
+        transcription_orchestrator = get_transcription_orchestrator(
+            diarization_mode=auth.diarization_mode, hf_token=hf_token
+        )
+        log.info(f"Transcription orchestrator initialized: diarization_mode={auth.diarization_mode}")
+
+        # Validate orchestrator is accessible
+        if await transcription_orchestrator.health_check():
+            log.info("Transcription orchestrator health check: ✔")
+        else:
+            log.warning("Transcription orchestrator health check failed")
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)